presto-iceberg/pom.xml

```diff
@@ -712,6 +715,7 @@
                             <requireUpperBoundDeps>
                                 <excludes combine.children="append">
                                     <exclude>com.google.guava:guava</exclude>
+                                    <exclude>commons-codec:commons-codec</exclude>
```

Comment:
Let's try to upgrade this dependency if AWS SDKs are bringing in a higher version.